workers/social/lib/social/models/invitation.coffee

```diff
@@ -218,7 +238,12 @@ module.exports = class JInvitation extends jraphical.Module
 
           async.parallel queue, (err, codes) ->
             return callback err  if err
-            return callback()  unless returnCodes
+
+            unless returnCodes
+              data = { type: 'create', invitations: codes }
+
+              return notifyGroupOnInvitationChange groupName, data, callback
```

Comment:
we need to store JInvitation instances to do any operation which is resend, revoke, etc. but we may get JInvitation instances by using email address then do operation. 
So you are right it may cause a problem especially sending huge invitations at the same time. I will update it soon.